registry/client/repository.go

```diff
@@ -292,9 +292,22 @@ func (t *tags) Get(ctx context.Context, tag string) (distribution.Descriptor, er
 	if err != nil {
 		return distribution.Descriptor{}, err
 	}
+
+	req, err := http.NewRequest("HEAD", u, nil)
+	if err != nil {
+		return distribution.Descriptor{}, err
+	}
+
+	for _, t := range distribution.ManifestMediaTypes() {
+		req.Header.Add("Accept", t)
+	}
```

Comment:
It looks like these headers should be included, but I'm not completely clear on how this is used by the proxy. Does the proxy use this method to get a manifest descriptor from the remote server? What does it use the manifest descriptor for?